src/uu/ls/src/ls.rs

```diff
@@ -1140,11 +1264,25 @@ only ignore '.' and '..'.",
                 .help("Print human readable file sizes (e.g. 1K 234M 56G).")
                 .overrides_with(options::size::SI),
         )
+        .arg(
+            Arg::new(options::size::KIBIBYTES)
+                .short('k')
+                .long(options::size::KIBIBYTES)
+                .help("default to 1024-byte blocks for file system usage; used only with -s and per directory totals"),
+        )
         .arg(
             Arg::new(options::size::SI)
                 .long(options::size::SI)
                 .help("Print human readable file sizes using powers of 1000 instead of 1024."),
         )
+        .arg(
+            Arg::new(options::size::BLOCK_SIZE)
+                .long(options::size::BLOCK_SIZE)
+                .takes_value(true)
+                .require_equals(true)
+                .value_name("BLOCK_SIZE")
+                .help("scale sizes by SIZE when printing them"),
```

Comment:
This should be `scale sizes by BLOCK_SIZE when printing them` because it will be displayed like this 

```shell
        --block-size=<BLOCK_SIZE>                    scale sizes by SIZE when printing them
```

Reply:
Thanks!  Great suggestion.  Would appreciate your giving the rest a look since you've thought about it yourself.  Sorry we got cross purposes on this implementation